includes/abstracts/abstract.llms.payment.gateway.php

```diff
@@ -26,15 +23,13 @@ abstract class LLMS_Payment_Gateway extends LLMS_Abstract_Options_Data {
 	 * Optional gateway description for the admin panel
 	 *
 	 * @var string
-	 * @since  3.0.0
```

Comment:
@gocodebox/engineering, I'm still confused. The documentation says class properties should have a changelog. Yet this PR removes single `@since` tags that document when the property was introduced. Are they **not** considered a changelog?

Here is a collection of all the documentation regarding class **properties** and changelogs.

> ## What should be documented
> The following elements should be documented using formatted documentation blocks (DocBlocks):
> + Class members (including properties and constants)

> ### Changelogs
> Whenever any code is changed within an element, an `@since`, `@version`, or `@deprecated` tag should be added to the element to document the change(s) which have been made.
> #### Changes Warrating a Changelog Entry
> Most code changes warrant a changelog entry to be recorded for the element but there are some exceptions.
> + Classes: Any breaking changes, deprecations, or the introduction of new class elements (elements which do not have their own changelog, such as class properties) require an accompanying `@since` tag entry. Changes to a class method should be recorded on the method's changelog, not on the class changelog.

> ## DocBlock Examples
> ### Class Members
> Class properties and constants should be formatted as follows:
> + Summary
> + Changelog
> + Type
> ```
> /**
>  * Summary.
>  *
>  * @since x.x.x
>  * @since x.x.x Description of member changes.
>  * @var type Optional description.
>  */